http/okhttp/src/main/java/com/okta/commons/http/okhttp/OkHttpRequestExecutor.java

```diff
@@ -126,7 +135,21 @@ public Response executeRequest(Request request) throws HttpException {
                 okRequestBuilder.head();
                 break;
             case POST:
-                okRequestBuilder.post(new InputStreamRequestBody(request.getBody(), request.getHeaders().getContentType()));
+                if(isMultipartFormDataForFileUploading) {
+                    String fileLocation = fetchHeaderValueAndRemoveIfExists(request, "x-fileLocation");
```

Comment:
Just for my understanding: why are we removing the headers `x-fileFormDataName` & `x-fileLocation` here?

Reply:
These headers are not required to perform file uploading. They are used to pass info from SDK to the HTTP client.